src/main/java/mate/controller/manufacturers/AddManufacturerController.java

```diff
@@ -0,0 +1,36 @@
+package mate.controller.manufacturers;
+
+import java.io.IOException;
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import mate.lib.Injector;
+import mate.model.Manufacturer;
+import mate.service.ManufacturerService;
+
+@WebServlet(urlPatterns = "/manufacturers/add")
+public class AddManufacturerController extends HttpServlet {
+    private static final Injector injector = Injector.getInstance("mate");
+    private static final ManufacturerService manufacturerService =
```

Comment:
Consider making the 'manufacturerService' field non-static. Static fields should be used with caution, especially when dealing with services or resources that might need to be cleaned up.